hack/local-up-cluster.sh

```diff
@@ -243,7 +243,13 @@ function start_apiserver {
 
 function start_controller_manager {
     CTLRMGR_LOG=/tmp/kube-controller-manager.log
-    sudo -E "${GO_OUT}/kube-controller-manager" \
+
+    enable_extensions=""
+   if echo "${RUNTIME_CONFIG}" | grep -q "extensions/v1beta1=true"; then
+      enable_extensions="--enable-experimental"
```

Comment:
You lost me here... I would expect that this should grep `extensions/v1beta1` (without the `=true`) and you should produce (for the controller) `--enable-experimental=true`.